src/main/java/com/mongodb/kafka/connect/source/MongoSourceTask.java

```diff
@@ -89,17 +91,20 @@
  */
 public class MongoSourceTask extends SourceTask {
     private static final Logger LOGGER = LoggerFactory.getLogger(MongoSourceTask.class);
+    private static final String INVALIDATE = "invalidate";
 
     private final Time time;
     private final AtomicBoolean isRunning = new AtomicBoolean();
+    private final AtomicBoolean isCopying = new AtomicBoolean();
 
     private MongoSourceConfig sourceConfig;
     private MongoClient mongoClient;
 
-    private final AtomicBoolean isCopying = new AtomicBoolean();
+    private boolean supportsStartAfter = true;
+    private boolean invalidatedCursor = false;
```

Comment:
Signifies the cursor has been invalidated - this can happen when a user matches only inserts but the db/coll was dropped anyway.